operator/src/main/java/oracle/kubernetes/weblogic/domain/model/ServerSpecCommonImpl.java

```diff
@@ -228,17 +228,20 @@ public V1SecurityContext getContainerSecurityContext() {
 
   @Override
   public String getDomainRestartVersion() {
-    return domainSpec.getRestartVersion();
+    // treat null, unset, and "" as the same restartVersion ""
```

Comment:
Need to see if you did it, but if we normalize missing labels on existing pods before creating the hash, then we won't roll existing pods.

Reply:
Isn't the hash generated using the above?   This change is for 3.0, so there won't be existing pods that aren't normalized.

Reply:
Do we still need these changes?

Reply:
The changes were never absolutely needed.  They're a "nice to have". 

The issue is that the average user plus at least one public Kubernetes API view 0-length and null strings as the same, but the operator treats them differently.  This makes the attribute a little confusing at its "edges".